CDFModule/Private/func_Get-TokenValues.ps1

```diff
@@ -25,6 +25,7 @@
             EnvNameId         = $CdfConfig.Application.Env.nameId
             EnvShortName      = $CdfConfig.Application.Env.shortName
             DomainName        = $CdfConfig.Domain.Config.domainName
+            DomainStorageName = $CdfConfig.Domain.ResourceNames.storageAccountName
```

Comment:
It would be good with a comment for this as a temporary/interim solution , a "TODO: remove when connections are being used for functions".